iotdb-core/node-commons/src/assembly/resources/conf/iotdb-system.properties.template

```diff
@@ -1867,6 +1867,38 @@ load_clean_up_task_execution_delay_time_seconds=1800
 # Datatype: int
 load_write_throughput_bytes_per_second=-1
 
+# Whether to enable the DataNode to actively listen for and load tsfile(default is enabled).
+# effectiveMode: hot_reload
```

Comment:
If the effective mode is "hot_reload", the parameters must be reloaded by "loadHotModifiedProps" and the code shall check the hot modified props and respond to their changes.